src/components/EmptyState/README.md

```diff
@@ -157,6 +157,11 @@ Use to explain a single feature before merchants have used it.
   heading="Manage your inventory transfers"
   action={{content: 'Add transfer'}}
   secondaryAction={{content: 'Learn more', url: 'https://help.shopify.com'}}
+  footer={
```

Comment:
I'd be tempted to create a new example that demonstrates the footer content so people can see what it looks like with and without the footer (and to drive home the idea that the footer is optional)